tests/markdown.test.js

```diff
@@ -0,0 +1,18 @@
+const Markdown = require('shared/naturalcrit/markdown.js');
+
+// FIXME: tests below are disabled, becaues marked.js uses replaceAll function,
+// which is only supported starting with node.js 15.0.0:
+// https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V15.md#v8-86---35415
+// Need to enable them back when we uplifit node.js requirements of the project
+
+test.skip('Escapes <script> tag', function() {
+	const source = '<script></script>';
+	const rendered = Markdown.render(source);
+	expect(rendered).toMatch('&lt;script&gt;&lt;/script&gt;');
+});
```

Comment:
Is there a way to run tests on text files? It would be handy to have a folder of inputs / outputs that we can compare like Marked.js does (https://github.com/markedjs/marked/tree/master/test/specs/new). This would keep the actual test script small and easy to navigate while allowing lots and lots of tests of all different sizes.

Reply:
I guess we can organize something like this for sure, but I haven't explored it